requirements.txt

```diff
@@ -1,10 +1,19 @@
-Django~=2.2.0
+certifi==2020.12.5
+chardet==3.0.4
+Django==2.2.12
```

Comment:
Please fix the Django versioning to use the latest version of 2.2, not restrict it to a single version.